ipaserver/install/server/replicainstall.py

```diff
@@ -1077,7 +1099,7 @@ def promote_check(installer):
     remote_api.finalize()
     installer._remote_api = remote_api
 
-    check_remote_version(remote_api)
+    check_remote_compatibility(remote_api)
```

Comment:
This check is executed only in replica promotion. There should be a check for classic replica install as well; IMO it should reject install when FIPS is enabled.